bundles/org.openhab.binding.revogismartstripcontrol/README.md

```diff
@@ -0,0 +1,68 @@
+# RevogiSmartStripControl Binding
+
+This binding is written to control the [Revogi Smart Power Strip](https://www.revogi.com/smart-power/smart-power-strip-eu/#section6). The device has 6 power plugs that can be switched independently, or all together.
+
+It was hard to find out how to control it without internet access, but there's a way to use UDP packets. See `LAN UDP Control.pdf` in the `doc` folder for details. This was the only document the Revogi support provided. 
```

Comment:
And you rename the pdf to not have spaces in the name? Kinda a pet peeve of mine.